source/reference/minio-mc-admin/mc-admin-user-svcacct-add.rst

```diff
@@ -84,18 +84,34 @@ Parameters
 .. mc-cmd:: --access-key
    :optional:
 
-   A string to use as the access key for this account. Omit to let MinIO autogenerate a random value.
+   A string to use as the access key for this account.
+   Omit to let MinIO autogenerate a random 20 character value.
 
    Access Key names *must* be unique across all users.
 
 .. mc-cmd:: --comment
    :optional:
 
-   .. versionadded:: RELEASE.2023-01-28T20-29-38Z
+   .. versionchanged:: RELEASE.2023-05-18T16-59-00Z
 
-   Add a note to the service account.
+   This option has been removed.
+   Use ``--description`` or ``--name`` instead.
+
+.. mc-cmd:: --description
+   :optional:
+
+   .. versionadded:: RELEASE.2023-05-18T16-59-00Z
+
+   Add a description for the service account.
    For example, you might specify the reason the service account exists.
 
+.. mc-cmd:: --name
+   :optional:
+
+   .. versionadded:: RELEASE.2023-05-18T16-59-00Z
+
+   Add a friendly name for the service account.
```

Comment:
This text matches what is in `--help`. 

Is it worth adding an example ("Test account for xyz team" ?) or is `friendly` sufficiently understood without additional context?

Reply:
Maybe
```suggestion
   Add a human-readable name for the service account.
```